File: year2020/month12/day1203/ms_1203.cpp
#include <iostream>

using namespace std;

long long solution(int a, int b);

int main() {
	int a;
	int b;

	cin >> a >> b;

	if ((-10000000 <= a && a <= 10000000) && (-10000000 <= b && b <= 10000000)) {
		cout << solution(a, b) << endl;
	}

	return 0;
}

long long solution(int a, int b) {
	long long answer = 0;
	int A = a;
	int B = b;

	if (A > B) {
		int temp = A;
		A = B;
		B = temp;
	}
	
	if (A != B) {
		for (int i = A; i <= B; i++) {
			answer += i;
		}
	}
	else answer = A;

	return answer;
}

File: year2020/month12/day1203/ms_1203.cs
using System;

namespace algorithm_114
{
    class ms_1203
    {
        static long solution(int a, int b) 
        {
            long answer = 0;
            int A = a;
            int B = b;

            if(A > B) 
            {
                int temp = A;
                A = B;
                B = temp;
            }

            if(A != B)
            {
                for(int i = A; i <= B; i++)
                {
                    answer += i;
                }
            }
            else answer = A;

            return answer;
        }

        static void Main(string[] args)
        {
            int a = Convert.ToInt32(Console.ReadLine());
            int b = Convert.ToInt32(Console.ReadLine());

            if((-10000000 <= a && a <= 10000000) && (-10000000 <= b && b <= 10000000))
            {
                Console.WriteLine(solution(a, b));
            }
        }
    }
}
